src/components/Auth/Login.js

```diff
@@ -1,6 +1,53 @@
+import axios from "axios";
 import React from "react";
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { useAuthContex } from "../../contex/auth-contex";
+
 import "./auth.css";
 const Login = () => {
+  const navigateHome = useNavigate();
+  const { authDispatch } = useAuthContex();
```

Comment:
Name it `useAuthCountext`, I know you are writing useAuthContext everywhere but it can cause trouble later on for you or may be for reviewer also as he might think that it is a type, write full names it is easy to understand the code then.